docs/user-guide/configuring.md

```diff
@@ -566,8 +566,14 @@ ESLint extends configurations recursively so a base configuration can also have
 The `rules` property can do any of the following to extend (or override) the set of rules:
 
 * enable additional rules
-* override default options for rules from base configurations
-* disable rules from base configurations
+* change an inherited rule's severity without changing its options:
+    * Base config: `"eqeqeq": [2, "allow-null"]`
```

Comment:
Can you use the "error", "warning", "off" notation instead of the numeric notation for severity?